### PR TITLE
Refactor/footer contd

### PR DESCRIPTION
**Pull request by ivonne-hernandez**
What (if any) features are you implementing?
- I fixed the footer so that it sticks to the bottom of the page and displays based on the `App` content height.
- I added a hover effect to the share link button.

Desktop view:
Screenshot 2024-05-30 at 4 26 37 PM

### CODE REVIEW

File: src/Components/Share/Share.css [hunk 1]
```diff
@@ -71,6 +71,10 @@ button {
   justify-content: flex-end;
 }
 
+.button-no-format:hover {
```
Comment (CalebPena) [inline src/Components/Share/Share.css:74]:
```suggestion
.button-no-format {
```

File: src/App.css [hunk 2]
```diff
@@ -6,6 +6,9 @@
 #root {
   overflow-x: hidden;
   position: relative;
+  min-height: 100svh;
```
Comment (CalebPena) [inline src/App.css:9]: What does `svh` do?

File: src/App.css [hunk 3]
```diff
@@ -19,6 +22,10 @@
   max-width: var(--main-max-width);
 }
 
+.App {
```
Comment (CalebPena) [inline src/App.css:25]:
```suggestion
.app {
```

File: src/Components/Footer/Footer.tsx [hunk 4]
```diff
@@ -6,16 +6,12 @@ import { FormattedMessage } from 'react-intl';
...
+  const { getReferrer } = context;
 
   return (
     <footer>
-      <Paper
-        elevation={0}
-        sx={{ width: '100%', backgroundColor: `var(--midBlue-color)`, marginTop: '5rem' }}
-        square={true}
-      >
+      <Paper elevation={0} sx={{ width: '100%', backgroundColor: `var(--midBlue-color)` }} square={true}>
```
Comment (CalebPena) [inline src/Components/Footer/Footer.tsx]:
```suggestion
      <Paper elevation={0} sx={{ width: '100%', backgroundColor: theme.midBlueColor }} square={true}>
```

**Review by CalebPena — APPROVED** [review]
LGTM, couple of optional style changes.

File: src/App.css (hunk 2 above)
Comment (ivonne-hernandez) [reply src/App.css:9]: https://www.youtube.com/watch?v=ru3U8MHbFFI

**Review by CalebPena — APPROVED** [review]

**Pull request merged**